### PR TITLE
Update PendingMessages.java

### PR DESCRIPTION
**Pull request by blackcat42**
Changed ImportMessagesTask's protected status variable to a Boolean to fit the refactoring of ProcessSms.java

### CODE REVIEW

**Comment by eyedol** [comment]
Thanks for the `PR` The build is failing. Complaining about incompatible types; https://travis-ci.org/ushahidi/SMSSync/builds/57650881#L1427. 

Also on https://github.com/blackcat42/SMSSync/blob/patch-2/smssync/src/main/java/org/addhen/smssync/fragments/PendingMessages.java#L471 we should return the call to `importMessages()` instead of the hard coded `return true` 

After the fix, push the changes to this branch blackcat42:patch-2. This PR should will be updated automatically.

**Comment by blackcat42** [comment]
Thanks for the guidance! New to Git and didn't realize I could update multiple files in one patch.

File: smssync/src/main/java/org/addhen/smssync/messages/ProcessSms.java [hunk 1]
```diff
@@ -213,7 +213,7 @@ public boolean filterByRegex(String message, String filterText) {
      *
      * @return int 0 for success, 1 for failure.
```
Comment (eyedol) [inline smssync/src/main/java/org/addhen/smssync/messages/ProcessSms.java]: Could you change the doc for this as well?

Comment (eyedol) [inline smssync/src/main/java/org/addhen/smssync/messages/ProcessSms.java]: Could you change the doc for this as well?

**Comment by eyedol** [comment]
Great. Looking good. Added a minor comment. Also if you don't mind, could you add yourself to the [contributors.json](https://github.com/ushahidi/SMSSync/blob/develop/contributors.json) list? It's sorted by last name.

Thanks for your contribution.

**Pull request merged**